### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by MattiaBonfanti-CS**
Pull request to address the tasks for assignment 1

### CODE REVIEW

**Review by LaurensKrudde — APPROVED** [review]
Looks good

File: app.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    responses:
+      200:
+        description: Some result
+    """
+    return {
+        "result": "Please use the /predict endpoint to predict the sentiment of a review text!",
+    }
+
+
+@app.route("/predict", methods=["POST"])
```
Comment (LaurensKrudde) [inline app.py:33]: Shouldn't this be a GET request?

File: app.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    res = {
+        "result": prediction_map[prediction],
+        "classifier": "GaussianNB",
+        "review": restaurant_review
+    }
+
+    log.info(res)
+
+    return res
```
Comment (LaurensKrudde) [inline app.py:86]: How can I test this currently?

Comment (DanDanBro) [reply app.py:86]: follow readme for running the service, http://localhost:8080/apidocs for the requests

**Review by DanDanBro — APPROVED** [review]
Looks good

File: app.py (hunk 1 above)
Comment (MattiaBonfanti-CS) [reply app.py:33]: The review gets sent in the body of the request, which needs to be a POST in this case.

**Pull request merged**